ui/tests/acceptance/oidc-auth-method-test.js

```diff
@@ -6,19 +6,12 @@ import { fakeWindow, buildMessage } from '../helpers/oidc-window-stub';
 import sinon from 'sinon';
 import { later, _cancelTimers as cancelTimers } from '@ember/runloop';
 
-module('Acceptance | logout auth method', function (hooks) {
+module('Acceptance | oidc auth method', function (hooks) {
   setupApplicationTest(hooks);
   setupMirage(hooks);
 
   hooks.beforeEach(function () {
     this.openStub = sinon.stub(window, 'open').callsFake(() => fakeWindow.create());
-  });
-  hooks.afterEach(function () {
-    this.openStub.restore();
-  });
-
-  // coverage for bug where token was selected as auth method for oidc and jwt
-  test('it should populate oidc auth method on logout', async function (assert) {
     this.server.post('/auth/oidc/oidc/auth_url', () => ({
```

Comment:
What do you think about also (or replacing) this coverage with oidc at a non-standard mount path (eg. `oidc-test`)?

Reply:
Do you mean adding a test that covers logging in using the _Mount path_ input?

Reply:
I think as far as this logout test goes only the previously selected auth method is re-hydrated in the dropdown. It doesn't look to me like _Mount path_ gets re-hyrdrated but I could be missing something.

Reply:
You're right, on logout with oidc I think expected behavior is that the mount path does not get filled in. I was talking about making sure the `auth_url` is generated correctly from the form, but that might be too complicated 🤔

Reply:
Let me know what you think of the last commit. I updated the test that uses the auth path tabs and returned `test-path` from the mounts request rather than `oidc` and then assert that the correct `auth_url` endpoint is hit.

Reply:
Great! Thanks for updating that 👍